### PR TITLE
5.x Fix I18n command locale path

### PR DESCRIPTION
**Pull request by dereuromark**
I noticed this in an edge case
I guess it is not often used, since mostly those are already defined.
Or this happens in tests, like for me:

```
The question asked was: <question>What is the path you would like to output?
[Q]uit</question>
[/data/work/devilbox/data/www/app/vendor/dereuromark/cakephp-translateresources/locales] > 
```

But ROOT is the only constant without trailing DS, so here it should always be ROOT . DS . ...

### CODE REVIEW

File: src/Command/I18nExtractCommand.php [hunk 1]
```diff
@@ -231,7 +231,7 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             $message = "What is the path you would like to output?\n[Q]uit";
             $localePaths = array_values(App::path('locales'));
             if (!$localePaths) {
-                $localePaths[] = ROOT . 'resources' . DIRECTORY_SEPARATOR . 'locales';
+                $localePaths[] = ROOT . DS . 'resources' . DIRECTORY_SEPARATOR . 'locales' . DIRECTORY_SEPARATOR;
```
Comment (ADmad) [inline src/Command/I18nExtractCommand.php]:
```suggestion
                $localePaths[] = ROOT . DIRECTORY_SEPARATOR . 'resources' . DIRECTORY_SEPARATOR . 'locales' . DIRECTORY_SEPARATOR;
```

Mixing `DS` and `DIRECTORY_SEPARATOR` isn't nice.

File: src/I18n/MessagesFileLoader.php [hunk 2]
```diff
@@ -172,7 +172,7 @@ public function translationsFolders(): array
 
         $localePaths = App::path('locales');
         if (!$localePaths && defined('ROOT')) {
-            $localePaths[] = ROOT . 'resources' . DIRECTORY_SEPARATOR . 'locales' . DIRECTORY_SEPARATOR;
+            $localePaths[] = ROOT . DS . 'resources' . DIRECTORY_SEPARATOR . 'locales' . DIRECTORY_SEPARATOR;
```
Comment (ADmad) [inline src/I18n/MessagesFileLoader.php]:
```suggestion
            $localePaths[] = ROOT . DIRECTORY_SEPARATOR . 'resources' . DIRECTORY_SEPARATOR . 'locales' . DIRECTORY_SEPARATOR;
```

**Comment by dereuromark** [comment]
Ops, wrong rebase
Resolved in https://github.com/cakephp/cakephp/commit/8fb34e72904b4aaf10ff89eb4905686697915a37
